lib/Echidna/Types/Campaign.hs

```diff
@@ -44,7 +44,8 @@ data CampaignConf = CampaignConf
 data CampaignEvent
   = TestFalsified !EchidnaTest
   | TestOptimized !EchidnaTest
-  | NewCoverage !Int !Int !Int
+  | TestSimplified !EchidnaTest
```

Comment:
This is fine, but I'm thinking, should we use the "shrink" vocabulary here so it's easier to figure out what's going on? Like `TestShrunk`?